### DISCUSSION TITLE
M5Stamp C3U Mate working perfectly as a Base Station

### DISCUSSION THREAD

**Discussion by GinAndBacon** [ESPresense/ESPresense#747] [General]
I know the [M5Stamp C3 Mate](https://shop.m5stack.com/products/m5stamp-c3-mate-with-pin-headers) is already listed but the [M5Stamp C3U Mate](https://shop.m5stack.com/products/m5stamp-c3u-mate-with-pin-headers) works great also as a Base Station. Just tested it out and it is super responsive, although it's exactly the same as the M5Stamp C3 Mate with the below difference (copy/paste from mstack.com). Just hold down the LED button when plugging in the USB C port when flashing it via the espresence website as base station and not the RST button due to differences outlined below (C3 may be the same, do not own one). Probably overkill but super tiny and works way better then a generic ESP32 board as a base station. C3U doesn't have CH9102, only Download / Serial while C3 has Download / JTAG / CDC Serial. They are identical outside no CH9102 and C3U has 14 IO's instead of 13 IO's on the C3. Not sure if this is the right place but general seems like the best fit.

_The M5Stamp C3U is a refreshed version of the M5Stamp C3 which removes the CH9102 USB to UART converter, instead relying on the ESP32-C3’s onboard USB functionality. The rest of the M5Stamp C3U features the same functionality and components as the original M5Stamp C3.

Enter the program download mode operation:
1. Long press the center button (G9) of STAMP C3U under power failure condition. (G9 is LED button, not RST button)_

**Comment by oliverboermel** [comment]
Hi i tried all Firmwares but none is working wich one is the right one?

**Comment by DTTerastar** [comment]
Added it to https://espresense.com/nodes
